appeals/web/src/server/views/appeals/documents/decision-letter-upload.njk

```diff
@@ -0,0 +1,59 @@
+{% extends "app/layouts/app.layout.njk" %}
+
+{% from 'app/components/file-uploader.component.njk' import fileUploader %}
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block pageTitle %}
+	{{ pageHeadingText }}
+{% endblock %}
+
+{%- block pageHeading -%}
+	<div class="govuk-grid-row">
+		<div class="govuk-grid-column-full">
+			<span class="govuk-caption-l govuk-!-margin-bottom-3">{{ caseInfoText }}</span>
+			<h1 class="govuk-heading-l govuk-!-margin-bottom-3">{{ pageHeadingText }}</h1>
+		</div>
+	</div>
+{%- endblock -%}
+
+{%- set backLinkUrl -%}
+	{{ backButtonUrl }}
+{%- endset -%}
+
+{% block pageContent %}
+
```

Comment:
Very minor, but it would be good to tidy up the empty lines and indentation here (although the latter could just be github's rendering)